modules/ROOT/pages/apply-env-settings.adoc

```diff
@@ -0,0 +1,10 @@
+= Apply Environments Settings
+
+When you change your environment's Partner Manager or Access Management settings, follow these steps to apply the changes to all your runtime applications:
```

Comment:
Please add examples of what kind of configurations we are referring to.
"When you change your environment's Partner Manager or Access Management settings **such as Mule runtime version, vCore configurations, ActiveMQ configurations**"